upath/implementations/cloud.py

```diff
@@ -46,11 +48,15 @@ def _sub_path(self, name):
         relative path to `self`.
         """
         sp = re.escape(self.path)
-        subed = re.sub(f"^({self._url.netloc})?/?({sp}|{sp[1:]})/?", "", name)
-        return subed
+        netloc = self._url.netloc  # type: ignore
```

Comment:
is the `type: ignore` comment needed here?

Reply:
I have removed these, but the `# type: ignore` comments were added to help pass a more strict run of mypy using `--check-untyped-defs`:

```
Type check the interior of functions without type annotations (inverse: --no-check-untyped-defs)
```